src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -61,6 +62,10 @@ public function getDefinition()
                     "<?php\nfunction sample(\$a=10,\$b=20,\$c=30) {}\nsample(1,  2);",
                     ['keep_multiple_spaces_after_comma' => true]
                 ),
+                new CodeSample(
+                    "<?php\nfunction sample(\$a=10,\n    \$b=20,\$c=30) {}\nsample(1,\n    2);",
+                    ['ensure_fully_multiline' => true]
```

Comment:
what is a value of setting up `keep_multiple_spaces_after_comma` (to any `true` or `false`) when `ensure_fully_multiline=true` is set ?

what is a difference between `ensure_fully_multiline=true keep_multiple_spaces_after_comma=true` and `ensure_fully_multiline=true keep_multiple_spaces_after_comma=false`
?